File: VeamIt-project/Android/veam31000287/veam31000287/src/main/java/co/veam/veam31000287/ConsoleSellAudioPostHandler.java
package co.veam.veam31000287;

import java.util.ArrayList;

/**
 * Created by veam on 11/7/16.
 */
public class ConsoleSellAudioPostHandler implements HandlePostResultInterface {
    SellAudioObject sellAudio ;
    AudioObject audio ;

    public SellAudioObject getSellAudio() {
        return sellAudio;
    }

    public void setSellAudio(SellAudioObject sellAudio) {
        this.sellAudio = sellAudio;
    }

    public AudioObject getAudio() {
        return audio;
    }

    public void setAudio(AudioObject audio) {
        this.audio = audio;
    }

    @Override
    public void handlePostResult(ArrayList<String> results) {
        //NSLog(@"%@::handlePostResult",NSStringFromClass(this.class])) ;
        if(results.size()  >= 2){
            //NSLog(@"count >= 2") ;
            String result = results.get(0) ;
            if(result.equals("OK")){
                String sellAudioId = results.get(1) ;
                String audioId = results.get(2) ;

                sellAudio.setId(sellAudioId) ;
                audio.setId(audioId) ;
            }
        }
    }
}

File: VeamIt-project/iOS/veam00000000/ConsoleSellAudioPostHandler.m
//
//  ConsoleSellAudioPostHandler.m
//  veam00000000
//
//  Created by veam on 11/5/15.
//  Copyright (c) 2015 veam. All rights reserved.
//

#import "ConsoleSellAudioPostHandler.h"

@implementation ConsoleSellAudioPostHandler

@synthesize sellAudio ;
@synthesize audio ;

- (void)handlePostResult:(NSArray *)results
{
    //NSLog(@"%@::handlePostResult",NSStringFromClass([self class])) ;
    if([results count] >= 2){
        //NSLog(@"count >= 2") ;
        NSString *result = [results objectAtIndex:0] ;
        if([result isEqualToString:@"OK"]){
            NSString *sellAudioId = [results objectAtIndex:1] ;
            NSString *audioId = [results objectAtIndex:2] ;
            
            [sellAudio setSellAudioId:sellAudioId] ;
            [audio setAudioId:audioId] ;
        }
    }
}

@end
